elfi/methods.py

```diff
@@ -76,28 +76,57 @@ def sample(self, n_samples, *args, **kwargs):
         """
         raise NotImplementedError("Subclass implements")
 
-    def _get_distances(self, n_samples):
-        """Run the all-accepting sampler.
+    def estimate_batches_needed(self, n_samples, accept_rate):
+        return self.estimate_proposals_needed(n_samples, accept_rate) // self.batch_size
+
+    def estimate_proposals_needed(self, n_samples, accept_rate):
+        """The returned estimate is divisible with self.batch_size"""
```

Comment:
Added additional explanation. What is by the way the minimal format for a docstring?